README.md

```diff
@@ -877,6 +877,51 @@ rmplan agent plan.yml --executor claude-code
 rmplan agent plan.yml --executor direct-call
 ```
 
+### Claude Code Executor: Interactive Tool Permissions
+
+The Claude Code executor includes an interactive permission system that allows you to control which tool invocations are automatically approved. When Claude attempts to use a tool during execution, you'll see a permission prompt with three options:
+
+- **Allow**: Permits this specific tool invocation only
+- **Disallow**: Denies this specific tool invocation
+- **Always Allow**: Permanently approves this tool (or command prefix for Bash tools) for automatic execution in future sessions
+
```

Comment:
Add notes about how to actually enable this via environment variable or in the rmplan config, since this is disabled by default.